src/chart-review/chart-review.component.tsx

```diff
@@ -89,175 +34,37 @@ export default function ChartReview(props: any) {
-    scrollToItem(selected);
-  }, [selected]);
-
-  const nextPage = () => {
-    navRef.current.scrollLeft =
-      navRef.current.scrollLeft + navRef.current.clientWidth;
-  };
-
-  const previousPage = () => {
-    navRef.current.scrollLeft =
-      navRef.current.scrollLeft - navRef.current.clientWidth;
-  };
-
-  const checkForOverflow = () => {
-    if (
-      navRef &&
-      navRef.current &&
-      navRef.current.scrollWidth - navRef.current.clientWidth > 0
-    ) {
-      if (!navHeaderOverflowed) {
-        setNavHeaderOverflowed(true);
-        dispatch("overflowed-changed");
-      }
-    } else {
-      if (navHeaderOverflowed) {
-        setNavHeaderOverflowed(false);
-        dispatch("overflowed-changed");
-      }
-    }
-  };
-  checkForOverflow();
   return (
-    <>
-      <div className={styles.navContainer}>
-        {(headerPagerState.pagerVisibility === PagerVisibility.LEFT ||
-          headerPagerState.pagerVisibility === PagerVisibility.BOTH) && (
-          <button
-            className={styles.leftHeaderPagination}
-            onClick={previousPage}
-          >
-            <svg
-              className="omrs-icon omrs-type-body-regular"
-              fill="var(--omrs-color-ink-medium-contrast)"
-            >
-              <use xlinkHref="#omrs-icon-arrow-back"></use>
-            </svg>
-          </button>
-        )}
-        <nav
-          onScroll={onNavHeaderScrolled}
-          ref={navRef}
-          className={styles.topnav}
-        >
-          <ul>
-            {navbarItems &&
-              navbarItems.map((item, index) => {
-                return (
-                  <li key={index} ref={navItemsRefs[index]}>
-                    <div
-                      className={`${
-                        index === selected ? styles.selected : styles.unselected
-                      }`}
-                    >
-                      <Link
-                        to={
-                          tabHistory[item.path.substr(1)] ||
-                          defaultPath + item.path
-                        }
-                      >
-                        <button
-                          className="omrs-unstyled"
-                          id={"nav_item" + index}
-                          onClick={() => setSelected(index)}
-                        >
-                          {t(item.label, item.label)}
-                        </button>
-                      </Link>
-                    </div>
-                  </li>
-                );
-              })}
-          </ul>
-          <VisitButton />
-        </nav>
-        {(headerPagerState.pagerVisibility === PagerVisibility.RIGHT ||
-          headerPagerState.pagerVisibility === PagerVisibility.BOTH) && (
-          <button className={styles.rightHeaderPagination} onClick={nextPage}>
-            <svg
-              className="omrs-icon omrs-type-body-regular"
-              fill="var(--omrs-color-ink-medium-contrast)"
+    <div>
```

Comment:
@denniskigen this nav bar is going to go away and be replaced with hamburger menu that is part of global nav. This might be the right time to introduce how this will happen, perhaps with guidance from @FlorianRappl .